lib/util/source-code.js

```diff
@@ -185,6 +185,26 @@ SourceCode.prototype = {
     },
 
     /**
+     * Retrieves an array containing all strings (" or ') in the source code.
```

Comment:
This object is part of the public API, and I'd rather not expose these methods. Can you move these into local functions in the rule?